extensions/objects/models/objects_test.py

```diff
@@ -15,13 +15,30 @@
 # limitations under the License.
 
 """Tests for typed object classes (mostly normalization)."""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import division  # pylint: disable=import-only-modules
+from __future__ import print_function  # pylint: disable=import-only-modules
 
 import inspect
+import os
+import sys
 
 from core.tests import test_utils
 from extensions.objects.models import objects
 import schema_utils_test
 
+_PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+_FUTURE_PATH = os.path.join(_PARENT_DIR, 'oppia_tools', 'future-0.17.1')
+
+sys.path.insert(0, _FUTURE_PATH)
+
+# pylint: disable=wrong-import-position
+# pylint: disable=wrong-import-order
+import past.builtins  # isort:skip
+
```

Comment:
Please remove this blank line and keep all new imports as tight as possible

Reply:
Done